examples/skipper-0.18/example.eskip

```diff
@@ -0,0 +1,22 @@
+hello:
+        Path("/hello")
+        -> status(200)
+        -> inlineContent("Hello, World!")
+        -> <shunt>;
+
+null:
+        Path("/null")
+        -> "http://172.32.255.1/null";
+
+gold:
+        Path("/gold")
+        -> "http://172.32.255.1/gold";
+
+oneb:
+        Path("/1b")
+        -> "http://172.32.255.1/1b";
+
```

Comment:
Any reason to leave these routes here? They don't work anyway (you get an `Unhandled page fault` if you try to curl them).